lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -8208,6 +8208,12 @@ const byte * InterpreterStackFrame::OP_ProfiledLoopBodyStart(const byte * ip)
         {
             BYTE* buffer = arr->GetBuffer();
             *(T2*)(buffer + index) = (T2)GetRegRaw<T2>(value);
+            return;
+        }
+        if (this->m_functionBody->IsWasmFunction())
+        {
+            //MGTODO : Change this to throw WebAssembly.RuntimeError once implemented
+            Js::Throw::FatalInternalError();
```

Comment:
You should throw a normal javascript runtime error for the time being so we can try/catch it in tests